types/edit-json-file/edit-json-file-tests.ts

```diff
@@ -3,6 +3,8 @@ import * as editJsonFile from 'edit-json-file';
 const jsonEditor: editJsonFile.JsonEditor = editJsonFile('1.json'); // $ExpectType JsonEditor
 jsonEditor.set('a', 2); // $ExpectType JsonEditor
 jsonEditor.set('b', []); // $ExpectType JsonEditor
+jsonEditor.append('a', 'b'); // $ExpectType JsonEditor
+jsonEditor.pop('a'); // $ExpectType any
```

Comment:
This returns `JsonEditor` in `index.d.ts` – which I confirmed in code of the library itself too – and is responsible for the CI failure.
```suggestion
jsonEditor.pop('a'); // $ExpectType JsonEditor
```